actions_install.sh

```diff
@@ -10,8 +10,6 @@ sudo apt-spy2 fix --commit
 # after selecting a specific mirror, we need to run 'apt-get update'
 sudo apt-get -o Acquire::Retries=3 update
 
-sudo apt-get -o Acquire::Retries=3 install -y libllvm8 -V
```

Comment:
dont remove these lines

Reply:
`libllvm8` is a direct dependency of `clang-format`, so it gets installed anyway. But I can change it.

Reply:
I added `libllvm` to the `apt-get`call for installing `clang-format` and `cppcheck`.